vaadin-confirm-dialog-flow-parent/vaadin-confirm-dialog-flow/src/main/java/com/vaadin/flow/component/confirmdialog/ConfirmDialog.java

```diff
@@ -695,6 +697,62 @@ public void setCloseOnEsc(boolean closeOnEsc) {
         getElement().setProperty("noCloseOnEsc", !closeOnEsc);
     }
 
+    /**
+     * Confirm dialog does not support replacing content.
+     * <p>
+     * This method is inherited from {@link HasOrderedComponents} and has been
+     * marked as deprecated to indicate that it is not supported.
+     *
+     * @deprecated since v24.4, not supported
+     */
+    @Deprecated
+    @Override
+    public void replace(Component oldComponent, Component newComponent) {
+        // NO-OP
```

Comment:
I wonder if these methods should still do what they did before rather than no-op to minimize the chance of breaking something.

Reply:
Might make sense. Even if it does not work as intended, it does something.

Reply:
Yeah, just to be on the safe side.

Reply:
Updated the PR to retain the old behaviour.